src/components/nav/NavBarComponent.jsx

```diff
@@ -22,21 +23,34 @@ const NavBarComponent = () => {
   return (
     <div>
       <Navbar color="dark" dark>
-        <NavbarBrand href="/">Bob's BurgerTown</NavbarBrand>
+        <NavbarBrand href="/">
+          <h1 className={styles.logo}>BurgerTown</h1>
+        </NavbarBrand>
 
-        <NavbarToggler onClick={isToggle} />
+        <NavbarToggler onClick={isToggle} className={styles.navbar_toggler} />
 
         <Collapse isOpen={isOpen} navbar>
           <Nav className="me-auto" navbar>
-            <NavItem>Item</NavItem>
             <UncontrolledDropdown nav>
-              {headers.map((header, i) => (
-                <DropdownToggle key={i} nav>
+              {headers.map((header, id) => (
+                <DropdownToggle nav>
                   <Link
+                    className={styles.nav_links}
+                    key={id}
                     onClick={() => setPath(header)}
                     to={`/categories/${header}`}
                   >
-                    {header}
+                    {header === "storeNextDoor" ? (
+                      <span>Store Next Door</span>
+                    ) : header === "characters" ? (
+                      <span>Characters</span>
+                    ) : header === "episodes" ? (
+                      <span>Episodes</span>
+                    ) : header === "pestControlTruck" ? (
+                      <span>Pest Control Truck</span>
+                    ) : header === "burgerOfTheDay" ? (
+                      <span>Burger Of The Day</span>
+                    ) : null}
```

Comment:
If you [create a map](https://softwareengineering.stackexchange.com/questions/307639/what-does-mapping-mean-in-programming#:~:text=In%20the%20most%20general%20sense%2C%20%22mapping%22%20in%20programming%20means%20taking%20several%20things%20and%20then%20somehow%20associating%20each%20of%20them%20with%20another%20thing.) of the headers like this:
```javascript
// since these values are static, no need to
// declare / maintain them in the component
const headersMap = {
  storeNextDoor: "Store Next Door",
  characters: "Characters",
  episodes: "Episodes",
  pestControlTruck: "Pest Control Truck",
  burgerOfTheDay: "Burger Of The Day",
};
```

Then you can render something like this:

```suggestion
                    <span>{headersMap[header] ?? <></>}</span>
```

 In `headersMap`, we're mapping the api value to the display value, which we can express generically like this:
```
map = {
  ApiKey: DisplayValue
}
```
Maps are a super powerful design concept.

The funky `??` above is called the [nullish coalescing operator ](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator), it's similar to the ternary operator but more simple, imo, even though the name is weird af

Reply:
This is way cleaner! I also think the ?? is easier. I tried it without the ?? <></> and it still rendered the same